lib/src/components/NxSegmentedButton/NxSegmentedButton.scss

```diff
@@ -77,34 +98,24 @@
   }
 
   &.nx-segmented-btn__main-btn {
-    // get fancy to prevent mitred corners on the inner border
-    border-top-style: none;
-    border-bottom-style: none;
-    border-left-style: none;
-
-    padding-top: var(--nx-spacing-2x);
-    padding-bottom: var(--nx-spacing-2x);
-    padding-left: var(--nx-spacing-6x);
-
-    &::after {
-      border-top-right-radius: 0;
-      border-bottom-right-radius: 0;
+    @include main-btn-styling;
+
+    &:focus:not(:active) {
+      &::after {
+        border-top-right-radius: 0;
```

Comment:
Can't these parts be included in the mixins as well?

Reply:
> Can't these parts be included in the mixins as well?

Though I guess there's no reason to move them there, since they don't get repeated

Reply:
> Can't these parts be included in the mixins as well?

I debated on that, and ultimately left them out because of the fact that the mixin is used for disabled buttons in dark mode as well and seemed odd to include it there (even if it doesn't actually affect disabled buttons)